com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumePerSceneData.cs

```diff
@@ -67,10 +58,23 @@ public void OnBeforeSerialize()
             }
         }
 
-        internal void StoreAssetForState(ProbeVolumeState state, ProbeVolumeAsset asset)
+#if UNITY_EDITOR
+        internal ProbeVolumeAsset CreateAssetForCurrentState()
+        {
+            var asset = ProbeVolumeAsset.CreateAsset(gameObject.scene, m_CurrentState);
+
+            assets[m_CurrentState] = asset;
+            QueueAssetLoading();
```

Comment:
This is not necessarily something we want. Why change the previous logic that will enqueue asset loading upon state change?

Reply:
If you are just talking about QueueAssetLoading i guess i can remove it since it's called after anyway during baking